.dockerignore

```diff
@@ -0,0 +1,4 @@
+Dockerfile
+*.md
+thirdeye-ui/
+
```

Comment:
let's add

```
**/target/
# tests are not run
**/src/test
examples
.idea
.github
doc
tmp
```

this helps `COPY ./ ./` to be fast, especially for a local host that also does local mvn build like our ci